gc/ogc/gx_struct.h

```diff
@@ -65,6 +65,7 @@ typedef struct _gx_rmodeobj {
 	u16 viWidth;
 	u16 viHeight;
 	u32  xfbMode;
+	u8  rgb;
```

Comment:
Could we do without this extra field, and assume that `VI_TVMODE_DEBUG_PROG` is always RGB?

Reply:
The EURGB60 modes are also all RGB -- you could do a manual check for those modes as well wherever `rgb` is used, but it felt more in spirit with the way the modes are laid out to add a field.